afs-storage-api/src/main/java/com/powsybl/afs/storage/events/ParentChanged.java

```diff
@@ -16,9 +16,17 @@
 
     public static final String TYPENAME = "PARENT_CHANGED";
 
+    @JsonProperty("newParentId")
+    private String newParentId;
+
     @JsonCreator
-    public ParentChanged(@JsonProperty("id") String id) {
+    public ParentChanged(@JsonProperty("id") String id, @JsonProperty("newParentId") String newParentId) {
         super(id, TYPENAME);
+        this.newParentId = newParentId;
```

Comment:
Is it possible that this object has no new parent? In that case, you should assert that it's not null.

Reply:
It is not possible for this event not to have a newParentId